### ISSUE TITLE
Add button to clean bookmarks description

### ISSUE THREAD

**Issue by Jean-BaptisteC** [organicmaps/organicmaps#9802]
**Is your feature request related to a problem? Please describe.**
Asked on french telegram channel
Users use description field on bookmark to store long text and when he want to remove all text, he need to press back button on keyboard to clean description

**Describe the ideal solution**
- Add button like for bookmark title to clean description in one tap and maybe to prevent miss click add dialog window to confirm clean the field

**Comment by biodranik** [comment]
1. Will there be an easy way to undo accidentally deleted text?
2. Isn't there already an option to "Select All" from the context menu and then delete?

Deletion of a potentially long description is a dangerous operation, and should not be done accidentally.

Is it possible to clarify the user about his/her use-cases for cleaning description often?

**Comment by map-per** [comment]
I don't think we need to implement something here. The user can mark / highlight the text and then delete everything at once
